yamlenc/yamlenc.py

```diff
@@ -65,9 +65,17 @@ def default(self):
 
     def lookup(self, nodename):
         """Look up ENC by nodename"""
+
+        # try exact match
+        if nodename in self.nodes:
```

Comment:
Fun fact, an "in" test is really slow in Python  ... but it won't matter until we have a few million nodes ;-)